### PR TITLE
♻️ Use objstr in Bento components

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

File: extensions/amp-inline-gallery/1.0/thumbnails.js [hunk 1]
```diff
@@ -69,7 +70,10 @@ export function Thumbnails({
 
   return (
     <BaseCarousel
-      className={`${className} ${classes.thumbnails}`}
+      className={objstr({
+        [classes.thumbnails]: true,
+        [className]: true,
```
Comment (dvoytenko) [inline extensions/amp-inline-gallery/1.0/thumbnails.js:75]: Same? `!!className`?

File: src/preact/component/contain.js [hunk 2]
```diff
@@ -76,7 +77,10 @@ function ContainWrapperWithRef(
     <Comp
       {...rest}
       ref={ref}
-      className={`${className || ''} ${wrapperClassName || ''}`.trim() || null}
+      className={objstr({
+        [wrapperClassName]: wrapperClassName,
+        [className]: className,
```
Comment (dvoytenko) [inline src/preact/component/contain.js:82]: `!!className`?

File: src/preact/component/wrapper.js [hunk 3]
```diff
@@ -41,7 +42,10 @@ function WrapperWithRef(
     <Comp
       {...rest}
       ref={ref}
-      className={`${className || ''} ${wrapperClassName || ''}`.trim() || null}
+      className={objstr({
+        [wrapperClassName]: wrapperClassName,
+        [className]: className,
```
Comment (dvoytenko) [inline src/preact/component/wrapper.js:47]: Same. `!!className`? Either way is ok, but let's make it consistent.

File: src/preact/component/wrapper.js [hunk 4]
```diff
@@ -41,7 +42,10 @@ function WrapperWithRef(
     <Comp
       {...rest}
       ref={ref}
-      className={`${className || ''} ${wrapperClassName || ''}`.trim() || null}
+      className={objstr({
+        [wrapperClassName]: wrapperClassName,
```
Comment (dvoytenko) [inline src/preact/component/wrapper.js:46]: `!!wrapperClassName`?

**Review by jridgewell — APPROVED** [review]

File: src/preact/component/contain.js [hunk 5]
```diff
@@ -76,7 +77,10 @@ function ContainWrapperWithRef(
     <Comp
       {...rest}
       ref={ref}
-      className={`${className || ''} ${wrapperClassName || ''}`.trim() || null}
+      className={objstr({
+        [wrapperClassName]: wrapperClassName,
```
Comment (caroqliu) [inline src/preact/component/contain.js:81]:
```suggestion
        [wrapperClassName]: !!wrapperClassName,
```

**Review by caroqliu — APPROVED** [review]

**Review by krdwan — APPROVED** [review]
LGTM

**Pull request closed without merging**